packages/eslint-plugin-office-addins/src/utils/utils.ts

```diff
@@ -61,8 +61,14 @@ function findOfficeApiReferencesInScope(scope: Scope): void {
       reference.resolved &&
       proxyVariables.has(reference.resolved)
     ) {
-      if (isLoadReference(reference.identifier)) {
+      const node: TSESTree.Node = reference.identifier;
+      if (isLoadReference(node)) {
         apiReferences.push({ operation: "Load", reference: reference });
+      } else if (
+        node.parent?.type === TSESTree.AST_NODE_TYPES.MemberExpression &&
```

Comment:
Why are we looking at the parent and not the reference itself?

Reply:
It is because of the tree itself. This is how we do it normally when calling the `findTopExpression`. I just continued using what we use.

Reply:
OK . . I'm forgetting my tree.  I thought the identifier was a CallExpression, but it's a type in and of itself (pointing to the actual property in question) and the parent is the CallExpression that is the 'property path' leading up to this one.  But by moving the 'findTobLevelExpression' into the method we can leave that method to worry about the CallExpression type.  Here we can just pass in the node to the method asking if this node is a method call.